pages/wiki/creating-an-asteroid-app.md

```diff
@@ -152,10 +151,88 @@ Change the following `Environment` variables:
 
 Your app should now be able to run from the application when you click the start button in the bottom left sidebar.
 
+# Learn QML
+
+Main docs: https://doc.qt.io/ (AOS uses qt5)
```

Comment:
Let's always refer to AsteroidOS as AsteroidOS and not AOS.

Also Qt takes a capital letter ;)